caper/caper_backend.py

```diff
@@ -180,6 +180,9 @@ def __init__(self, db_type=None, db_timeout=None,
 class CaperBackendGCP(dict):
```

Comment:
Beside the point of this PR, but if you don't have a really good reason you should never subclass `dict` and use `collections.UserDict` instead. For more information, see Trey Hunner's blogpost https://treyhunner.com/2019/04/why-you-shouldnt-inherit-from-list-and-dict-in-python/

Reply:
I see. There are two more branches (PIP-937, PIP-881) to be merged and they are based on this branch (PR). one of them has modification on this `caper_backend.py`. I am not sure if git rebase/auto-merge can deal with this beautifully. I will correct this in the latest PR (PIP-937).